package main

import "testing"

func TestReplyHandler(t *testing.T) {
	sum := 1
	if sum != 1{
		t.Errorf("test test failed")
	}
}
